lib/AppInfo/Application.php

```diff
@@ -19,9 +19,11 @@ public function __construct(array $urlParams = []) {
 
 	public function register(IRegistrationContext $context): void {
 		$context->registerCapability(Capabilities::class);
+		$context->registerDashboardWidget(DashboardWidget::class);
 	}
 
 	public function boot(IBootContext $context): void {
+		\OCP\Util::addStyle('notes', 'global');
```

Comment:
Dashboard has an event which you can use instead of loading your css all the time (even on api requests you make it calculate paths)

Reply:
I need this style also for unified search. Is there also an event over there?